consensus/consensus-types/src/block_data.rs

```diff
@@ -98,11 +98,6 @@ impl BlockData {
     }
 
     pub fn round(&self) -> Round {
-        // Round numbers:
-        // - are reset to 0 periodically.
-        // - do not exceed std::u64::MAX - 2 per the 3 chain safety rule
-        // (ConsensusState::commit_rule_for_certified_block)
-        assumed_postcondition!(self.round < std::u64::MAX - 1);
```

Comment:
The assumption should instead be checked here: https://github.com/libra/libra/blob/1860e532734ec181b50124391cc67ce44bce6a48/consensus/safety-rules/src/safety_rules.rs#L107